### ISSUE TITLE
Implement live preview of changes made in CSS variables collection

### ISSUE THREAD

**Issue by warpech** [Starcounter/Blending#117]
### Expected result

As a user of BlendingEditor, while editing a CSS Variables Collection that is bound the currently displayed layout, I would expect the changes that I make to the values to be immediately reflected, even before I hit the "Save" button.

### Actual result

Currently, the value changes are applied only after the "Save" button is pressed.

**Comment by warpech** [comment]
Duplicate of https://github.com/Starcounter/Blending/issues/108
